webapp/src/client.ts

```diff
@@ -271,7 +271,8 @@ export default class CallsClient extends EventEmitter {
             this.intervalID = window.setInterval(() => {
                 this.ws?.send('reaction', {
                     data: JSON.stringify({
-                        emoji: emojis[count],
```

Comment:
We can remove this now since https://github.com/mattermost/mattermost-plugin-calls/pull/199 is open

Reply:
Done in https://github.com/mattermost/mattermost-plugin-calls/pull/198/commits/e13e727f1f9063d553bb6691c65938815104638a